README.md

```diff
@@ -87,9 +87,8 @@ Each group (except common) should be installed on same system.
 
 ### Common
 
-* `java::default` - Uses [JAVA cookbook][java-cookbook].
-Installs [JAVA JDK][java-jdk-7] required by other AET components.
-Please notice that Karaf for AET works with JAVA ver. 1.7 only.
+* `java::default` - Installs [Open JDK 8][java-openjdk-8] required by other AET components.
```

Comment:
I'm wondering if we should still go with openjdk and not Oracle? 
Oracle JDK8 is currently still available publicly and it works great on both linux and windows

Reply:
How is it about accepting Oracle terms (cookie). Isn't it that JAVA cookbook forces users to set the attribute to true, because the license needs to be accepted explicite?

I mean: i thought that there is some licensing requirement for everyone that uses Oracle JAVA - to accept Oracle terms and conditions. So if we make it default, someone could have Oracle JAVA without intentionally accepting the conditions.

That is why I decided to set the Oracle JAVA attributes in Vagrantfile.

What do you think? Maybe I am too careful here?